plugins/docker-machine/_docker-machine

```diff
@@ -330,7 +330,7 @@ _docker-machine() {
     _arguments -C \
         "(- :)"{-h,--help}"[Show help]" \
         "(-D --debug)"{-D,--debug}"[Enable debug mode]" \
-        '(-s --stroage-path)'{-s,--storage-path}'[Configures storage path]:file:_files' \
+        '(-s --storage-path)'{-s,--storage-path}'[Configures storage path]:file:_files' \
```

Comment:
Based on the preceding line, this seems likely to be correct, but I can't state with any confidence what this changes.